GiEnJul/ClientApp/src/routes/registerGiver/SummaryRegistration.tsx

```diff
@@ -41,92 +91,160 @@ const SummaryRegistration: React.FC<Props> = ({ nextStep, prevStep, submit, valu
-                //onChange={handlefullnameChange}
-                />
-                <TextField
-                    disabled
-                    variant="outlined"
-                    margin="normal"
-                    required
-                    fullWidth
-                    id="email"
-                    label="Epost"
-                    name="email"
-                    autoComplete="email"
-                    autoFocus
-                    value={values.email}
-                // onChange={handleEmailChange}
-                />
-                <TextField
-                    disabled
-                    variant="outlined"
-                    margin="normal"
-                    required
-                    fullWidth
-                    name="phoneNumber"
-                    label="Telefonnummer"
-                    id="phoneNumber"
-                    autoComplete="tel"
-                    value={values.phoneNumber}
-                // onChange={handleTlfChange}
-                />
-                <TextField
-                    disabled
-                    variant="outlined"
-                    margin="normal"
-                    required
-                    fullWidth
-                    name="maxRecievers"
-                    label="Familiesammensetning"
-                    id="maxRecieverse"
-                    autoComplete="tel"
-                    value={values.familyType}
-                // onChange={handleTlfChange}
-                />
-            </FormControl>
-            <Grid container spacing={2} justify="center" className={classes.submit}>
+                <Grid container className={classes.inputRow} >
+                    <Grid item xs={9}>
+                        <SelectValidator
+                            disabled={changeLocation}
+                            variant="outlined"
+                            fullWidth
+                            placeholder={values.location}
+                            label="Lokasjon*"
+                            name="location-input"
+                            value={values.location ? values.location : ""}
+                            id="location-input"
+                            onChange={handleChangeLocation}
+                            errorMessages={['Vennligst velg lokasjon']}
+                        >
+                            {LOCATIONS.map(x =>
+                                <MenuItem key={x} value={x}>{x}</MenuItem>)}
+                        </SelectValidator>
+                    </Grid>
+                    <Grid item xs={3}>
+                        <Button onClick={handleChangeLocation}>
+                            <EditOutlinedIcon className={classes.iconSelector}></EditOutlinedIcon>
+                        </Button>
+                    </Grid>
+                </Grid>
+                <Grid container className={classes.inputRow} >
+                    <Grid item xs={9}>
+                        <TextValidator
```

Comment:
The InputValidator from #139 should be used instead of the TextValidator class. This way we handle all validation from one single component.